tools/starman.psgi

```diff
@@ -6,7 +6,8 @@ BEGIN {
 
 package LedgerSMB::FCGI;
 
-use lib 'lib';
+use FindBin;
+use lib $FindBin::Bin . "/../lib";
```

Comment:
@einhverfr did you test this? Is this running off the script's directory, or off Starman's directory? (the FindBin::Bin directory, I mean).

My intuition is that this doesn't work the way you intend it to.